source/Fleet.h

```diff
@@ -87,9 +78,11 @@ class Fleet {
 	const Government *government = nullptr;
 	const Phrase *names = nullptr;
 	const Phrase *fighterNames = nullptr;
-	std::vector<Variant> variants;
+	std::vector<std::pair<const Variant *, int>> stockVariants;
```

Comment:
Can you add a bit of comments on the stockVariants and on the variants variable to explain the difference?

Reply:
The comment adds quite some clarity, but it still sounded a little bit confusing on what stockVariants really is.
I think something like the following text is even more clear:
```
// StockVariants contains references to the named root-node variants (as defined
// in GameData) that are referenced from this fleet definition. The variants vector
// contains the local variants that were defined by this fleet definition.
```